ratex-popup/popup.js

```diff
@@ -85,7 +85,7 @@ function popupInit(initialTime = 10000, followUpTime = 10000, duration = 5000) {
       "<div class=textContainer>" +
       "<div class=closeContainer>" +
       "<div id=" + popup.titleId + "></div>" +
-      "<img src='https://raw.githubusercontent.com/rate-engineering/ratex-webflow-snippets/44ee8793cc84d126b9ecf0f36bdc9f4dd1c2bfbf/ClosePopup.svg' alt='close' id=" + popup.closeId + " />" +
+      "<div id=" + popup.closeId + ">X</div>" +
```

Comment:
how about just use inline <svg>, something like https://codepen.io/acusti/pen/PwzBZv, just change the color a bit

Reply:
The purpose of this feature is to give convenience towards the users as much as possible. Hence with few lines of codes to display out the feature. I had worked on implementing the svg via javascript